base/src/test/resources/success/src/Assoc.aya

```diff
@@ -71,10 +71,10 @@ def infixl |=> (a b : Dir) : Sig Dir ** Dir => (a, b)
 
 def id {A : Type} (a : A) : A => a
 
-def secR => (|=> id id id id R)
+def secR => ( |=> id id id id R)
 def secL => (id L |=>)
 
-def secR' => (|=> R)
+def secR' => ( |=> R)
 def secL' => (L |=>)
```

Comment:
The reformat seems unnecessary since spaces are not inserted after `|=>`.

Reply:
Ohhhh. Then we should replace the test case with a better name IMO

Reply:
Oh no